extensions/gdx-freetype/src/com/badlogic/gdx/graphics/g2d/freetype/FreeType.java

```diff
@@ -362,14 +598,41 @@ public ByteBuffer getBuffer() {
 		 * @return Pixmap representing the glyph, needs to be disposed manually.
 		 */
 		public Pixmap getPixmap(Format format) {
-			Pixmap pixmap = new Pixmap(getWidth(), getRows(), Format.Alpha);
-			BufferUtils.copy(getBuffer(), pixmap.getPixels(), pixmap.getPixels().capacity());
-			Pixmap converted = new Pixmap(pixmap.getWidth(), pixmap.getHeight(), format);
-			Blending blending = Pixmap.getBlending();
-			Pixmap.setBlending(Blending.None);
-			converted.drawPixmap(pixmap, 0, 0);
-			Pixmap.setBlending(blending);
-			pixmap.dispose();
+			return getPixmap(format, Color.WHITE);
+		}
+
+		public Pixmap getPixmap(Format format, Color color) {
+			int width = getWidth();
+			ByteBuffer src = getBuffer();
+			Pixmap pixmap;
+			if (color == Color.WHITE) {
+				pixmap = new Pixmap(width, getRows(), Format.Alpha);
+				BufferUtils.copy(src, pixmap.getPixels(), pixmap.getPixels().capacity());
+			} else {
+				pixmap = new Pixmap(width, getRows(), Format.RGBA8888);
+				int srcPitch = getPitch();
+				int srcRGBA = Color.rgba8888(color);
+				IntBuffer dst = pixmap.getPixels().asIntBuffer();
+				for (int y = 0; y < getRows(); y++) {
+					int ySrcPitch = y * srcPitch;
+					int yWidth = y * width;
+					for (int x = 0; x < width; x++) {
+						//use the color value of the foreground color, blend alpha
+						byte alpha = src.get(ySrcPitch + x);
+						dst.put(yWidth + x, (srcRGBA & 0xffffff00) | (int)((srcRGBA & 0xff) * (alpha & 0xff)/255f));
+					}
+				}
+			}
+
+			Pixmap converted = pixmap;
+			if (format != Format.RGBA8888) {

```

Comment:
Actually I'm not an expert here, I've just followed the suggestion of @badlogic:
>If someone can provide a PR that fixes this (do it natively **or use the old impl given default parameters**) we can merge it.

To use the old implementation in case of default color, and it was previously implemented as Alpha, don't know why actually :(